conftest.py

```diff
@@ -390,20 +390,33 @@ def remove_upgrade_tests_based_on_config(
 
 def filter_deprecated_api_tests(items: list[Item], config: Config) -> list[Item]:
     # filter out deprecated api tests, if explicitly asked or if running upgrade/install tests
-    items_to_return: list[Item] = []
     if config.getoption("--skip-deprecated-api-test") or config.getoption("--install") or config.getoption("--upgrade"):
```

Comment:
should we also add `--upgrade_custom` here?

Reply:
yeah, I think so. Unless you want deprecated api test running there

Reply:
I don't need deprecated api in custom upgrade lanes :)
I think I just missed this part when implemented custom upgrade